ngraph/frontend/onnx/frontend/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 
 set(TARGET_NAME onnx_ov_frontend)
 
-set(ONNX_OPSET_VERSION 13 CACHE INTERNAL "Supported version of ONNX operator set")
+set(ONNX_OPSET_VERSION 15 CACHE INTERNAL "Supported version of ONNX operator set")
```

Comment:
Why do we need to change opset version?

Reply:
This is to avoid a warning that models containing ops with opset 15 are not supported. We've already switched to ONNX 1.10 and this change should have been added in that PR but I forgot about it.